flowmachine/flowmachine/core/query.py

```diff
@@ -777,6 +752,14 @@ def invalidate_db_cache(self, name=None, schema=None, cascade=True, drop=True):
         """
         with rlock(self.redis, self.md5):
             con = self.connection.engine
+            try:
+                table_form = self.get_table()
+                if table_form is not self:
```

Comment:
And what is a possible scenario where `table_form` is _not_ `self`?

Reply:
```python
dl = daily_location("2016-01-01").store().result()
dl_table = dl.get_table()
```

`dl` is a query object, which happens to be cached and hence exists as a table, `dl_table` is a `Table` object which points _to_ that cached table.